src/engine/ui/widgets/Slider.hxx

```diff
@@ -67,11 +75,14 @@ private:
   /// minimum slider value
   int m_minVal = 0;
   /// maximum slider value
-  int m_maxVal = 100;
+  int m_maxVal = 0;
```

Comment:
why zero? the default from 0 - 100 makes sense. values for specific usecases should be set via a setter function (that's not yet implemented).

i'd suggest `setRange(int min, int max);`

Reply:
I was assuming that the slider was for moving other elements, in which case there would be no usecase for setting an arbitrary maxval.